src/Components/Homepage.jsx

```diff
@@ -0,0 +1,13 @@
+import { Link } from "react-router-dom";
+
+export default function Homepage() {
+    return (
+        <>
```

Comment:
Rather than wrapping this in a react fragment <></> you would want to wrap this in a semantic tag, this page currently wouldn't have a <main> tag